testkit-backend/backend.go

```diff
@@ -98,9 +98,9 @@ func (g GenericTokenManager) GetAuthToken(_ context.Context) (neo4j.AuthToken, e
 	return g.GetAuthTokenFunc(), nil
 }
 
-func (g GenericTokenManager) OnTokenExpired(_ context.Context, token neo4j.AuthToken) error {
-	g.OnTokenExpiredFunc(token)
-	return nil
+func (g GenericTokenManager) HandleSecurityException(_ context.Context, token neo4j.AuthToken, securityException *db.Neo4jError) (bool, error) {
+	g.HandleSecurityExceptionFunc(token, securityException)
+	return false, nil
```

Comment:
Feels like this should not just return `false` but actually ask TestKit what to return.

Reply:
updated 👍